compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/codegen/JvmConstants.java

```diff
@@ -357,6 +357,8 @@ public class JvmConstants {
                                                           DEFAULT_VERSION);
     public static final String BUILT_IN_PACKAGE_NAME = "lang" + ENCODED_DOT_CHARACTER + "annotations";
     public static final String MODULE_START_ATTEMPTED = "$moduleStartAttempted";
+    public static final String MODULE_START_PARENT_ATTEMPTED = "$moduleStartParentAttempted";
```

Comment:
It is used to stop duplicate moduleStart() invocations being executed. In cases of a common dependency, the `moduleStart()` of the child will be invoked from both parent packages. Since `$moduleStartParentAttempted` will be set to true in the first invocation, the method won't be executed on the duplicate invocation. 

We had to use the field name `$moduleStartParentAttempted` because `$moduleStartAttempted` was already in use.

Reply:
Ack. It would be more clear to have `$parentModuleStartAttempted`.

Reply:
Changed the field name in 48458c0b7392851701057030520eed36632d904f.